### PR TITLE
[coinbase-v2.2.0-alpha.1]: Upgrade coinbase wallet sdk

### PR DESCRIPTION
**Pull request by cb-jake**
### Description
upgrade coinbase wallet sdk v3.6.0

### Checklist
- [x] The version field in `package.json` is incremented following [semantic versioning](https://semver.org/)
- [x] The box that allows repo maintainers to update this PR is checked
- [x] I tested locally to make sure this feature/fix works
- [x] I have run `yarn check` & `yarn build` to confirm there are not any associated errors
- [ ] This PR passes the Circle CI checks

### CODE REVIEW

File: packages/coinbase/yarn.lock [hunk 1]
```diff
@@ -0,0 +1,1968 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (Adamj1232) [inline packages/coinbase/yarn.lock:1]: @cb-jake can you please remove this yarn.lock ad it is not required

File: packages/coinbase/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/coinbase",
-  "version": "2.1.3",
+  "version": "2.2.0",
```
Comment (Adamj1232) [inline packages/coinbase/package.json:3]:
```suggestion
  "version": "2.2.0-alpha.1",
```
This will allow for publishing of an alpha package for testing

**Review by Adamj1232 — CHANGES_REQUESTED** [review]
@cb-jake thanks for the PR! a couple small comment. Can you also remove the packages/coinbase/yarn.lock file as its not required Thanks!

**Comment by cb-jake** [comment]
Thanks @Adamj1232. I addressed your comments in [5615ce9](https://github.com/blocknative/web3-onboard/pull/1373/commits/5615ce911dd544ce5e0375bdf62916a718df28f9)

**Comment by Adamj1232** [comment]
@cb-jake we had a PR merged to fix the entry point for the coinbase SDK in certain envs. If you could please pull in the latest develop and push we can get this PR merged tomorrow. Thank you!

**Comment by cb-jake** [comment]
Thanks @Adamj1232! I just rebased. LMK if anything else is needed!

File: packages/coinbase/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/coinbase",
-  "version": "2.1.4-alpha.1",
+  "version": "2.2.1-alpha.1",
```
Comment (Adamj1232) [inline packages/coinbase/package.json:3]:
```suggestion
  "version": "2.2.0-alpha.1",
```

**Comment by Adamj1232** [comment]
> Thanks @Adamj1232! I just rebased. LMK if anything else is needed!

Great one small note on the PR. We will try and get an alpha version out this afternoon. Thanks again!

**Review by Adamj1232 — APPROVED** [review]
Thank you for this update, its much appreciated!!

**Comment by cb-jake** [comment]
Thanks again @Adamj1232!

**Pull request merged**